specs/src/specs/params.md

```diff
@@ -64,5 +64,6 @@ are blocked by the `x/paramfilter` module.
 | staking.MaxValidators                         | 100                                         | Maximum number of validators.                                                                                                                                                                   | True                      |
 | staking.MinCommissionRate                     | 0.05 (5%)                                   | Minimum commission rate used by all validators.                                                                                                                                                 | True                      |
 | staking.UnbondingTime                         | 1814400 (21 days)                           | Duration of time for unbonding in seconds.                                                                                                                                                      | False                     |
+| minfee.GlobalMinGasPrice                      | 0.002                                        | All transactions must have a gas price greater than or equal to this value                                                                                                                             | True                      |
```

Comment:
[nit] this param table is alphabetically sorted. Can you please move `minfee` up?

Reply:
[tangent] we don't have to figure this out in this PR but I wonder if it would make sense to include the app version that a constant is applicable for in this table. For example, I think some of these params could be modified to be different in different app versions and at that point it's not immediately obvious if this table should reflect the original or updated value. Perhaps both with a new column for app version?

Reply:
yes, i can move it in a follow up pr. thanks @rootulp 

and to you second point i think that's a good call. I'll export your comment as an issue.